main.go

```diff
@@ -151,6 +149,63 @@ func sanitize(str string) string {
 	return strings.Join(ret, "\n")
 }
 
+// execute custom tests for specific challenges
+// such as those which have many solutions
+func customTester(challengeID string, solution string) string {
+	if challengeID == "desafio-13" {
+		if validKnights(solution) {
+			return "ok"
+		}else{
+			return "not ok"
+		}
+	}else {
+		return solution
+	}
+}
+
+// check for a valid solution for knight's tour
+func validKnights(solution string) bool{
+	const Steps = 64 // must be 64 for a 8x8 table
+	sl := strings.Split(solution, "\n")
+	var prevStep string = sl[0]
+	var row,col int
+
+	// a valid solution must have 64 steps
+	if len(sl) != Steps {return false}
+	// a valid solution must not have duplicated entries
+	//var test string
+	for i, step := range sl {
+		for j, test_step := range sl{
```

Comment:
Não use underscores em variavels em go (no caso, `test_step`). Também, simplifique pra `step` apenas (se possível).

Reply:
Eu tinha percebido essa característica do Go, mas acho que deixei passar essa variável.  Infelizmente não deu pra simplificar porque esse "for" já está dentro de outro for que usa um contador step.